cmd/node/main.go

```diff
@@ -301,3 +307,11 @@ func attachFileLogger(log logger.Logger, flagsConfig *config.ContextFlagsConfig)
 
 	return fileLogging, nil
 }
+
+func checkHardwareRequirements() error {
+	if !cpuid.CPU.Supports(cpuid.SSE4, cpuid.SSE42) {
```

Comment:
I do not like having these hardcoded, can we somehow extract them in config.toml ?